docs/overview.md

```diff
@@ -0,0 +1,65 @@
+# Overview
+
+A [Svelte](https://svelte.dev/) Single-Page App with Server-Side Rendering (SSR).
+## Feature Summary
+
+|    Feature     |                  URL                   |
+| -------------- | -------------------------------------- |
+| Dashboard      | `/`                                    |
+| Source Library | `/library/*/*`                         |
+| Note Library   | `/notes/*/*`, `/notes/*/*/:id`         |
+| Reader UI      | `/sources/:storageId/:sourceId/:path+` |
+| Reader Profile |                                        |
```

Comment:
> I usually do pull requests to 'dev' instead of 'master',

Ah, good to know. I've updated the base branch for this PR.

I'll make the changes you have noted. Thanks!

Reply:
@Marie000 I've added the missing features to the table in commit 5827b13. I've also added a description and simplified the routing. This stuff tends to get out of date -- I think the simplified route mapping in the table is probably more resilient, leaving the code as the source of truth.